### DISCUSSION TITLE
Line spacing

### DISCUSSION THREAD

**Discussion by robertsonj3** [SixLabors/ImageSharp#2656] [Q&A]
I'm trying to draw text with extra line spacing between paragraphs using RichTextOptions.
I am aware of the LineSpacing option, but this applies to all lines, I just want it at the line breaks.

**Comment by JimBobSquarePants** [comment]
Yeah, line spacing operates on all lines (as it does in other libraries and word processors/browsers). What you need to do is insert additional breaks into the input string.

I can make that easier by exposing the enumerator to ensure you can capture them all to augment. Could you raise an issue for that.
